expr/expr.go

```diff
@@ -135,17 +134,78 @@ func (e *Meta) unmarshal(data []byte) error {
 // Masq (Masquerade) is a special case of SNAT, where the source address is
 // automagically set to the address of the output interface. See also
 // https://wiki.nftables.org/wiki-nftables/index.php/Performing_Network_Address_Translation_(NAT)#Masquerading
-type Masq struct{}
+type Masq struct {
+	Random      bool
+	FullyRandom bool
+	Persistent  bool
+	ToPorts     bool
+	RegProtoMin uint32
+	RegProtoMax uint32
+}
 
 func (e *Masq) marshal() ([]byte, error) {
+	msgData := []byte{}
+	if !e.ToPorts {
+		flags := uint32(0)
+		if e.Random {
+			flags |= 0x4
```

Comment:
Can you use the named constants instead of these values please? :)

Reply:
done